pydantic/networks.py

```diff
@@ -363,6 +363,13 @@ def import_email_validator() -> None:
         import email_validator
     except ImportError as e:
         raise ImportError('email-validator is not installed, run `pip install pydantic[email]`') from e
+    if hasattr(email_validator, '__version__') and getattr(email_validator, '__version__', '').partition('.')[0] == '2':
```

Comment:
```suggestion
    if getattr(email_validator, '__version__', '').partition('.')[0] == '2':
```
With `''` as a default value, `''.partition('.')` returns `('', '', '')`

Reply:
which is fine - 
v2.1 has __version__ and will pass
v1.x/v2.0 lack __version__ and the test below distinguishes in v1.x and v2.0 by testing for `normalized`.

Reply:
Yes, what I meant is that the `hasattr` check is not necessary because of the `getattr` fallback to `''`. But I think it's fine to keep it for clarity